어드민 > 점주 페이지 완성

jiwonns opened the pull request:
## #️⃣연관된 이슈

> #35  

## 📝작업 내용

> 운영자 페이지 점주 부분 개발

### 스크린샷 (선택)


## 💬리뷰 요구사항(선택)

>

jiwonns committed:
Merge branch 'admin' of https://github.com/SHDS-ESC/Sol-Food into jiwon/admin/manage/user-management

# Conflicts:
#	src/main/java/configuration/MvcConfig.java

jiwonns committed:
Merge branch 'jiwon/admin/manage/user-management' of https://github.com/SHDS-ESC/Sol-Food into jiwon/admin/manage/user-management

# Conflicts:
#	src/main/java/configuration/MvcConfig.java

jiwonns committed:
Refactor: Test Config scan 경로 변경

jiwonns committed:
Feat: user 페이지 네이션 초안

jiwonns committed:
Merge branch 'admin' of https://github.com/SHDS-ESC/Sol-Food into jiwon/admin/manage/user-management

jiwonns committed:
Merge branch 'admin' of https://github.com/SHDS-ESC/Sol-Food into admin

jiwonns committed:
Feat: user payment 초안 작성

jiwonns committed:
Feat: Admin 브랜치 동기화

jiwonns committed:
Feat: Admin 브랜치 동기화

jiwonns committed:
Merge branch 'jiwon/admin/manage/user-management' of https://github.com/SHDS-ESC/Sol-Food into admin

jiwonns committed:
Merge branch 'jiwon/admin/manage/payment-management' of https://github.com/SHDS-ESC/Sol-Food into jiwon/admin/manage/user-management

jiwonns committed:
Feat: 유저 목록 페이지네이션 적용

10,20,50개씩보기 적용

jiwonns committed:
Feat: 관리자 로그인 페이지 구현

관리자 로그인 구현

jiwonns committed:
Feat: 날아간 파일 복구

jiwonns committed:
Merge branch 'jiwon/admin/manage/user-management' of https://github.com/SHDS-ESC/Sol-Food into admin

# Conflicts:
#	src/main/java/kr/co/solfood/admin/home/AdminHomeController.java
#	src/main/webapp/WEB-INF/views/admin/user-management/home.jsp

jiwonns committed:
Merge branch 'jiwon/admin/manage/user-management' into admin

# Conflicts:
#	src/main/java/kr/co/solfood/admin/home/AdminHomeController.java

jiwonns committed:
Feat: 날아간 파일 복구

jiwonns committed:
Merge branch 'jiwon/admin/manage/user-management' of https://github.com/SHDS-ESC/Sol-Food into admin

jiwonns committed:
Feat: Owner 상태 업데이트 기능 추가

오너의 상태를 업데이트 해주는 기능 추가

jiwonns committed:
Fix: 상태 변경 후 새로고침 시 셀렉터 표기 오류 수정

5y1ee commented:
pr 제목이 헷갈리네요
점주 페이지라길래 owner 브랜치 인줄..

jiwonns commented:
@5y1ee 헉, 그러네요! 고쳐 놓겠습니다!

sngsngUDON commented on src/main/java/kr/co/solfood/admin/login/AdminLoginController.java:
```diff
@@ -1,33 +1,40 @@
+    public void login() {
+    }
+
+    @PostMapping("/login")
```
혹시 이 부분에서 LoginService 를 2번 호출하는 것 같은데 한번만 호출해서 이 부분을 처리할 수는 없나요 ??

sngsngUDON replied:
혹시 그리고 이곳에서 try-catch 를 사용하는 것은 어떠신지 ,,,,

jiwonns replied:
변수에 담아서 쓰는 걸로 바꿔보겠습니다!

jiwonns replied:
try - catch로 바꾸겠습니당

sngsngUDON commented on src/test/java/kr/co/solfood/admin/AdminTest.java:
test 코드까지,,,, 수고하셨습니다 :) 저도 참고해서 test 코드 다시 도전을,,

jiwonns replied:
넵, 감사합니다 !

sngsngUDON commented on src/main/java/kr/co/solfood/admin/home/AdminHomeController.java:
```diff
@@ -65,4 +63,16 @@ public String ownerManagement(Model model) {
+
+    @ResponseBody
+    @GetMapping("/home/payment-management/status-update")
+    public String OwnerStatusUpdate(@RequestParam("ownerId") long ownerId, @RequestParam("status") String status) {
```
혹시 이부분은 ownerStatusUpdate 가 맞지 않나요 ...?

jiwonns replied:
헉 맞네요! 수정하겠습니다

sngsngUDON reviewed (commented):
admin 부분 작업을 안해서 처음보는 부분이 많았네요 ㅎㅎ;; 최대한 리뷰해보려고 노력했습니다

jiwonns committed:
Merge branch 'admin' of https://github.com/SHDS-ESC/Sol-Food into jiwon/admin/manage/owner-management

jiwonns committed:
Fix: (#68) 리뷰 내용 반영

> 함수명 OwnerStatusUpdate -> ownerStatusUpdate로 변경

> 로그인 컨트롤러 try-catch, 변수로 선언하여 재사용

jiwonns commented:
@sngsngUDON 리뷰 내용 수정 반영하였습니다! 확인 후 머지 부탁드릴게요!
